spmd/tensor/ops/pointwise_ops.py

```diff
@@ -118,12 +118,30 @@
 # ]
 
 pointwise_ops = [
-    "aten.relu.default",
+    "aten._to_copy.default",
+    "aten._local_scalar_dense.default",
+    "aten.add.Tensor",
+    "aten.add.Scalar",
+    "aten.add_.Tensor",
+    "aten.add_.Scalar",
+    "aten.add.out",
+    "aten.abs.default",
+    "aten.bitwise_and.Tensor",
+    "aten.bitwise_and_.Tensor",
+    "aten.bitwise_or_.Tensor",
+    "aten.eq.Tensor",
     "aten.gelu.default",
+    "aten.relu.default",
+    "aten.le.Tensor",
```

Comment:
Can we also add relu backward which is needed for ViT.

https://github.com/pytorch/tau/pull/382/files#diff-a6184a94d36719b27f3b3b5c3c4efd24261d7dc8830ed45e96eac330dbed5824R124

Thanks!

Reply:
Yeah sure we can add it in the following PRs, would like to keep this PR succinct to disable fallback mode.